test/browser/utils.tsx

```diff
@@ -113,13 +125,21 @@ export async function copySelectedCell() {
   });
 }
 
+export function copySelectedCellNew() {
+  return userEvent.keyboard('{Control>}c');
```

Comment:
Shouldn't we stop pressing ctrl?
```suggestion
  return userEvent.keyboard('{Control>}c{/Control}');
```